docs/Makefile

```diff
@@ -73,6 +73,10 @@ singlehtml:
 	@echo
 	@echo "Build finished. The HTML page is in $(BUILDDIR)/singlehtml."
 
+.PHONY: livehtml
+livehtml:
```

Comment:
Unrelated to this PR, but it's an handy target to have.

Reply:
What is "livehtml" vs the `doc-html` target?

Reply:
got it! What if we change the name to `doc-html-live` (to tie it to the other target), and add a comment explaining the difference in the behavior?

Reply:
`doc-html` is in the main Makefile. `livehtml` is just part of the docs Makefile.